my-sms/spec/controllers/students_controller_spec.rb

```diff
@@ -23,119 +25,115 @@
 # removed from Rails core in Rails 5, but can be added back in via the
 # `rails-controller-testing` gem.
 
-RSpec.describe StudentsController, :type => :controller do
-
-  # This should return the minimal set of attributes required to create a valid
-  # Student. As you add validations to Student, be sure to
-  # adjust the attributes here as well.
-  let(:valid_attributes) {
-    skip("Add a hash of attributes valid for your model")
-  }
-
-  let(:invalid_attributes) {
-    skip("Add a hash of attributes invalid for your model")
-  }
-
-  # This should return the minimal set of values that should be in the session
-  # in order to pass any filters (e.g. authentication) defined in
-  # StudentsController. Be sure to keep this updated too.
-  let(:valid_session) { {} }
-
-  describe "GET #index" do
-    it "returns a success response" do
-      student = Student.create! valid_attributes
-      get :index, {}, valid_session
-      expect(response).to be_success
+RSpec.describe StudentsController, type: :controller do
+  let(:student) { FactoryBot.create(:student) }
```

Comment:
Minor one, but you can add some config to your rails helper to automatically include the FactoryBot methods: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#rspec

After which, this line becomes:

```ruby
let(:student) { create(:student) }
```

Reply:
Added in a1a9ea184bcabc7ce5d1917ec246dbdae2e1f7e0